.travis/install.sh

```diff
@@ -8,26 +8,34 @@ pushd common/ && pip install -e . && popd
 pushd pulpcore/ && pip install -e . && popd
 pushd plugin/ && pip install -e .  && popd
 
-export COMMIT_MSG=$(git log --format=%B -n 1 HEAD)
+export COMMIT_MSG=$(git show HEAD^2 -s)
```

Comment:
FYI, there's a variable for this I believe. Something like $TRAVIS_COMMIT_MESSAGE.

Reply:
There is, but it's for the merge commit of the PR into the branch it's going to. So it only says "Merge of blah into bleh". I have to find the parent commit of that.

Reply:
Git has an option for ignoring merge commits ```--no-merges```.  You could try that.